shared/src/main/scala/effekt/context/Context.scala

```diff
@@ -50,6 +52,10 @@ abstract class Context(val positions: Positions)
   // bring the context itself in scope
   implicit val context: Context = this
 
+  val DEFAULT_LLVM_VERSION: String = "12"
+  var LLVM_VERSION: String = DEFAULT_LLVM_VERSION
```

Comment:
You can add these options to `Config` and default them to "12".

Reply:
I hard-coded `"12"` in 0419025ca996340cf6e7370b6c4e43913f269edc for now.